components/Icon/Account.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Account = ({ width, height, colorStroke, colorFill }) => (
```

Comment:
Pasemos los default Proptypes a values default acá, asi podemos eliminar los proptypes.

```suggestion
const Account = ({ width = 36, height = 36, colorStroke = '#FFF', colorFill = '#067df7'}) => (
```